includes/class-wc-stripe-helper.php

```diff
@@ -439,73 +439,33 @@ public static function get_settings( $method = null, $setting = null ) {
 	 * List of legacy payment method classes.
 	 *
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in future versions.
 	 */
 	public static function get_legacy_payment_method_classes() {
-		$payment_method_classes = [
-			WC_Gateway_Stripe_Alipay::class,
-			WC_Gateway_Stripe_Bancontact::class,
-			WC_Gateway_Stripe_Boleto::class,
-			WC_Gateway_Stripe_Eps::class,
-			WC_Gateway_Stripe_Giropay::class,
-			WC_Gateway_Stripe_Ideal::class,
-			WC_Gateway_Stripe_Multibanco::class,
-			WC_Gateway_Stripe_Oxxo::class,
-			WC_Gateway_Stripe_P24::class,
-			WC_Gateway_Stripe_Sepa::class,
-		];
-
-		/** Show Sofort if it's already enabled. Hide from the new merchants and keep it for the old ones who are already using this gateway, until we remove it completely.
-		 * Stripe is deprecating Sofort https://support.stripe.com/questions/sofort-is-being-deprecated-as-a-standalone-payment-method.
-		 */
-		$sofort_settings = get_option( 'woocommerce_stripe_sofort_settings', [] );
-		if ( isset( $sofort_settings['enabled'] ) && 'yes' === $sofort_settings['enabled'] ) {
-			$payment_method_classes[] = WC_Gateway_Stripe_Sofort::class;
-		}
-
-		return $payment_method_classes;
+		return [];
 	}
 
 	/**
 	 * List of legacy payment methods.
 	 *
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in future versions.
```

Comment:
Nit: Another version fix.
```suggestion
	 * @deprecated 10.2.0 This method will be removed in future versions.
```